requirements.txt

```diff
@@ -54,3 +54,4 @@ SQLAlchemy==1.3.0
 sqlsoup==0.9.1
 urllib3==1.24.3
 Werkzeug==0.14.1
+flask-versioned==0.9.4.post20101221
```

Comment:
This code is from 2010?

Is there a newer possibiliy? I think new versions of flask actually come with components and features, that had to be added with extenions earlier. I would be curious, if there is a newer possibility.